build-linux/build.sh

```diff
@@ -14,6 +14,8 @@ mkdir godot
 cd godot
 tar xf /root/godot.tar.gz --strip-components=1
 
+dnf install -y binutils
+
```

Comment:
I'm adding this to the base image in https://github.com/godotengine/build-containers/pull/150 so this won't be needed once new images are made.